### PR TITLE
SDP-1055 - [Docs] Update README to reflect changes introduced in multi-tenancy

### PR DESCRIPTION
**Pull request by marwen-abid**
### What
Update README and database diagrams to reflect changes introduced in multi-tenancy. 

### Why
https://stellarorg.atlassian.net/browse/SDP-1055

#### PR Structure

* [x] This PR has a reasonably narrow scope (if not, break it down into smaller PRs).
* [x] This PR title and description are clear enough for anyone to review it.
* [x] This PR does not mix refactoring changes with feature changes (split into two PRs otherwise).

#### Thoroughness

* [x] This PR adds tests for the new functionality or fixes.
* [x] This PR contains the link to the Jira ticket it addresses.

#### Configs and Secrets

* [x] No new CONFIG variables are required -OR- the new required ones were added to the helmchart's [`values.yaml`] file.
* [x] No new CONFIG variables are required -OR- the new required ones were added to the deployments ([`pr-preview`], [`dev`], [`demo`], `prd`).
* [x] No new SECRETS variables are required -OR- the new required ones were mentioned in the helmchart's [`values.yaml`] file.
* [x] No new SECRETS variables are required -OR- the new required ones were added to the deployments ([`pr-preview secrets`], [`dev secrets`], [`demo secrets`], `prd secrets`).

#### Release

* [x] This is not a breaking change.
* [x] **This is ready for production.**. If your PR is not ready for production, please consider opening additional complementary PRs using this one as the base. Only merge this into `develop` or `main` after it's ready for production!

#### Deployment

* [ ] Does the deployment work after merging?

[`values.yaml`]: ../helmchart/sdp/values.yaml
[`pr-preview`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/common-previews/stellar-disbursement-platform/backend-helm-values
[`dev`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/backend-helm-values
[`demo`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/demo/demo-backend-helm-values
[`pr-preview secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/common-previews/externalsecrets-common-previews.yaml#L241-L346
[`dev secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/stellar-disbursement-platform-externalsecrets.yaml
[`demo secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/demo/demo-sdp-externalsecrets.yaml

#### PR Structure

* [ ] This PR has a reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR title and description are clear enough for anyone to review it.
* [ ] This PR does not mix refactoring changes with feature changes (split into two PRs otherwise).

#### Thoroughness

* [ ] This PR adds tests for the new functionality or fixes.
* [ ] This PR contains the link to the Jira ticket it addresses.

#### Configs and Secrets

* [ ] No new CONFIG variables are required -OR- the new required ones were added to the helmchart's [`values.yaml`] file.
* [ ] No new CONFIG variables are required -OR- the new required ones were added to the deployments ([`pr-preview`], [`dev`], [`demo`], `prd`).
* [ ] No new SECRETS variables are required -OR- the new required ones were mentioned in the helmchart's [`values.yaml`] file.
* [ ] No new SECRETS variables are required -OR- the new required ones were added to the deployments ([`pr-preview secrets`], [`dev secrets`], [`demo secrets`], `prd secrets`).

#### Release

* [ ] This is not a breaking change.
* [ ] **This is ready for production.**. If your PR is not ready for production, please consider opening additional complementary PRs using this one as the base. Only merge this into `develop` or `main` after it's ready for production!

#### Deployment

* [ ] Does the deployment work after merging?

[`values.yaml`]: ../helmchart/sdp/values.yaml
[`pr-preview`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/common-previews/stellar-disbursement-platform/backend-helm-values
[`dev`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/backend-helm-values
[`demo`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/demo/demo-backend-helm-values
[`pr-preview secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/common-previews/externalsecrets-common-previews.yaml#L241-L346
[`dev secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/stellar-disbursement-platform-externalsecrets.yaml
[`demo secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/demo/demo-sdp-externalsecrets.yaml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ make go-install
 stellar-disbursement-platform --help
 ```
 
+> [!WARNING]  
+> We're in the process of releasing a new multi-tenant version of the SDP. Develop branch currently contains the release candidate of multi-tenant SDP. 
+> For the stable single-tenant version, please use the latest `v1.x.x` tag.
```
Comment (reecexlm) [inline README.md:37]: do we want to link the release tags here? eg https://github.com/stellar/stellar-disbursement-platform-backend/releases

**Comment by reecexlm** [comment]
added some comments but feel free to merge or ping me for additional review if needed!

File: README.md [hunk 2]
```diff
@@ -141,35 +149,120 @@ In future iterations of the project, the Transaction Submission Service will pro
...
+
+```sh
+  EVENT_BROKER_TYPE: "KAFKA"
+  BROKER_URLS: # comma separated list of broker urls
+  CONSUMER_GROUP_ID: # consumer group id
+  KAFKA_SECURITY_PROTOCOL: # possible values "PLAINTEXT", "SASL_SSL", "SASL_PLAINTEXT" or "SSL"
+  KAFKA_SASL_USERNAME: # username for SASL authentication. Required if KAFKA_SECURITY_PROTOCOL is "SASL_SSL" or "SASL_PLAINTEXT"
+  KAFKA_SASL_PASSWORD: # password for SASL authentication. Required if KAFKA_SECURITY_PROTOCOL is "SASL_SSL" or "SASL_PLAINTEXT"
+  KAFKA_SSL_ACCESS_KEY: # access key (keystore) in PEM format. Required if KAFKA_SECURITY_PROTOCOL is "SSL" or "SASL_SSL"
+  KAFKA_SSL_ACCESS_CERTIFICATE: # certificate in PEM format that matches the access key. Required if KAFKA_SECURITY_PROTOCOL is "SSL" or "SASL_SSL"
```
Comment (CaioTeixeira95) [inline README.md:240]: Here is only if it's `SSL`

File: README.md [hunk 3]
```diff
@@ -141,35 +149,120 @@ In future iterations of the project, the Transaction Submission Service will pro
...
+
+**1. Jobs**
+
+> [!NOTE]  
+> Certain jobs are not listed here because they cannot be configured and are necessary to the functioning of the SDP. 
+
+* `send_receiver_wallets_sms_invitation_job`: This job is used to send disbursement invites to recipients. Its interval is configured through the `SCHEDULER_RECEIVER_INVITATION_JOB_SECONDS` environment variable.
+* `payment_to_submitter_job`: This job is used to submit payments from Core to the TSS. Its interval is configured through the `SCHEDULER_PAYMENT_JOB_SECONDS` environment variable.
+* `payment_from_submitter_job`: This job is used to notify Core that a payment has been completed. Its interval is configured through the `SCHEDULER_PAYMENT_JOB_SECONDS` environment variable.
+* `patch_anchor_platform_transactions_completion`: This job is used to patch transactions in Anchor Platform once payments reach a final state. Its interval is configured through the `SCHEDULER_PAYMENT_JOB_SECONDS` environment variable.
```
Comment (CaioTeixeira95) [inline README.md:254]: Perhaps we could add what "final state" means.

```suggestion
* `patch_anchor_platform_transactions_completion`: This job is used to patch transactions in Anchor Platform once payments reach a final state (`SUCCESS` or `FAILED`). Its interval is configured through the `SCHEDULER_PAYMENT_JOB_SECONDS` environment variable.
```

File: docs/images/multi-tenant-architecture.png
Comment (CaioTeixeira95) [inline docs/images/multi-tenant-architecture.png:1]: 🔥

**Review by CaioTeixeira95 — APPROVED** [review]
LGTM! That's an amazing work! 👏

File: README.md (hunk 1 above)
Comment (marwen-abid) [reply README.md]: addressed in 1b836460e43e6fc0fc48184e4626f9186ebec034

File: README.md (hunk 2 above)
Comment (marwen-abid) [reply README.md]: addressed in 1b836460e43e6fc0fc48184e4626f9186ebec034

File: README.md (hunk 3 above)
Comment (marwen-abid) [reply README.md]: addressed in 1b836460e43e6fc0fc48184e4626f9186ebec034

**Comment by reecexlm** [comment]
LGTM

**Pull request merged**